### ISSUE TITLE
Add "show/hide" for search result items

### ISSUE THREAD

**Issue by bnvk** [TransparencyToolkit/LookingGlass#25]
This was lost in refactoring but should be added back! This should allow the user to show / hide certain fields in a search result items. The point being- to make it easier to comb through results visually

**Comment by Shidash** [comment]
Going to close this for now as I think probably not a priority in the near future. But eventually we do need to add support for arbitrary structured datasets (csv and the like) so perhaps we should do this then. In general, offering a table-like view and a view where people can show/hide and control sort of fields may be worthwhile at some point. But most likely not until we add better structured dataset support.

What probably is useful in meantime is adding a length limit for category fields/show more button so that facets added by Catalyst don't take up lots of space.
